[2022-09-18 18:37:31,223] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [queued]>
[2022-09-18 18:37:31,645] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [queued]>
[2022-09-18 18:37:31,654] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:37:31,662] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 18:37:31,672] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 18:37:33,762] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-16T03:00:00+00:00
[2022-09-18 18:37:34,059] {standard_task_runner.py:51} INFO - Started process 431 to run task
[2022-09-18 18:37:34,213] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-16T03:00:00+00:00', '--job-id', '306', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpuozo6zcb']
[2022-09-18 18:37:34,272] {standard_task_runner.py:76} INFO - Job 306: Subtask write_query_big_query_table
[2022-09-18 18:37:36,185] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 18:37:37,055] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-16T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-16T03:00:00+00:00
[2022-09-18 18:37:37,266] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 18:37:37,589] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 18:37:40,327] {bigquery.py:1510} INFO - Inserting job airflow_1663526260234496_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 18:37:47,559] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526260234496_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663526260234496_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 18:37:47,740] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220916T030000, start_date=20220918T183731, end_date=20220918T183747
[2022-09-18 18:37:49,236] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 19:42:15,955] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [queued]>
[2022-09-18 19:42:16,009] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [queued]>
[2022-09-18 19:42:16,011] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:42:16,013] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 19:42:16,014] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 19:42:16,105] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-16T03:00:00+00:00
[2022-09-18 19:42:16,137] {standard_task_runner.py:51} INFO - Started process 4172 to run task
[2022-09-18 19:42:16,194] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-16T03:00:00+00:00', '--job-id', '436', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmpu0gh2ad5']
[2022-09-18 19:42:16,213] {standard_task_runner.py:76} INFO - Job 436: Subtask write_query_big_query_table
[2022-09-18 19:42:16,524] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 19:42:16,843] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-16T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-16T03:00:00+00:00
[2022-09-18 19:42:16,870] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 19:42:16,999] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 19:42:17,157] {bigquery.py:1510} INFO - Inserting job airflow_1663530137152926_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 19:42:18,732] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663530137152926_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663530137152926_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 19:42:18,805] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220916T030000, start_date=20220918T194215, end_date=20220918T194218
[2022-09-18 19:42:19,212] {local_task_job.py:118} INFO - Task exited with return code 1
[2022-09-18 20:14:10,491] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [queued]>
[2022-09-18 20:14:10,771] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [queued]>
[2022-09-18 20:14:10,777] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 20:14:10,779] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 20:14:10,780] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 20:14:11,091] {taskinstance.py:1038} INFO - Executing <Task(BigQueryOperator): write_query_big_query_table> on 2022-09-16T03:00:00+00:00
[2022-09-18 20:14:11,157] {standard_task_runner.py:51} INFO - Started process 913 to run task
[2022-09-18 20:14:11,265] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'bigquery_dag', 'write_query_big_query_table', '2022-09-16T03:00:00+00:00', '--job-id', '462', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_bigquery.py', '--cfg-path', '/tmp/tmp1rb74dcs']
[2022-09-18 20:14:11,322] {standard_task_runner.py:76} INFO - Job 462: Subtask write_query_big_query_table
[2022-09-18 20:14:13,372] {logging_mixin.py:103} INFO - Running <TaskInstance: bigquery_dag.write_query_big_query_table 2022-09-16T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 20:14:14,737] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=bigquery_dag
AIRFLOW_CTX_TASK_ID=write_query_big_query_table
AIRFLOW_CTX_EXECUTION_DATE=2022-09-16T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-16T03:00:00+00:00
[2022-09-18 20:14:14,783] {bigquery.py:680} INFO - Executing: 
        select * from githubarchive.day.20220914 where actor.id=66800817
            
[2022-09-18 20:14:15,163] {logging_mixin.py:103} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:2052 DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2022-09-18 20:14:16,027] {bigquery.py:1510} INFO - Inserting job airflow_1663532055981176_4e8e27051e4b7eda1ea1c14d0ba14e9a
[2022-09-18 20:14:20,555] {taskinstance.py:1396} ERROR - 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663532055981176_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1300, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 706, in execute
    encryption_configuration=self.encryption_configuration,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 2186, in run_query
    job = self.insert_job(configuration=configuration, project_id=self.project_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/common/hooks/base_google.py", line 425, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1512, in insert_job
    job.result()
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/query.py", line 1159, in result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/cloud/bigquery/job/base.py", line 631, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/google/api_core/future/polling.py", line 134, in result
    raise self._exception
google.api_core.exceptions.Forbidden: 403 Access Denied: Table githubarchive:day.20220914: User does not have permission to query table githubarchive:day.20220914, or perhaps it does not exist in location northamerica-northeast1.

(job ID: airflow_1663532055981176_4e8e27051e4b7eda1ea1c14d0ba14e9a)

                       -----Query Job SQL Follows-----                       

    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
   1:
   2:        select * from githubarchive.day.20220914 where actor.id=66800817
   3:            
    |    .    |    .    |    .    |    .    |    .    |    .    |    .    |
[2022-09-18 20:14:20,667] {taskinstance.py:1440} INFO - Marking task as UP_FOR_RETRY. dag_id=bigquery_dag, task_id=write_query_big_query_table, execution_date=20220916T030000, start_date=20220918T201410, end_date=20220918T201420
[2022-09-18 20:14:21,311] {local_task_job.py:118} INFO - Task exited with return code 1
